bin/node/runtime/src/lib.rs

```diff
@@ -49,7 +49,7 @@ use sp_runtime::traits::{
 	self, BlakeTwo256, Block as BlockT, StaticLookup, SaturatedConversion,
 	ConvertInto, OpaqueKeys, NumberFor,
 };
-use sp_version::RuntimeVersion;
+pub use sp_version::RuntimeVersion;
```

Comment:
```suggestion
use sp_version::RuntimeVersion;
```

Please include `sp-version` into `node-benching` directly.